p/sncb/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const {readFileSync} = require('fs')
+const {join} = require('path')
+const {Agent} = require('https')
+const products = require('./products')
+
+// `www.belgianrail.be:443` doesn't provide the necessary CA certificate
+// chain for Node.js to trust the certificate, so we manually add it.
+// todo: fix this properly, e.g. by letting them know
+const ca = readFileSync(join(__dirname, 'digicert-sha2-secure-server-ca.crt.pem'))
+const agent = new Agent({ca})
+const transformReq = (ctx, req) => ({...req, agent})
+
+const transformReqBody = (ctx, body) => {
+	body.client = {type: 'IPH', id: 'SNCB', name: 'sncb', v: '4030200'}
+	body.ver = '1.16'
+	body.auth = {type: 'AID', aid: 'sncb-mobi'}
+
+	return body
+}
+
+const sncbProfile = {
+	locale: 'de-BE', // todo: or `fr-BE`, or `nl-BE`?
```

Comment:
I think it makes sense to pick either `fr` or `nl` here, I'd argue for french since the profile is also called `sncb` (rather than `nmbs`).